overlord/snapstate/snapmgr_test.go

```diff
@@ -238,7 +246,7 @@ func (s *snapmgrTestSuite) TestInstallIntegration(c *C) {
 	defer s.state.Unlock()
 
 	chg := s.state.NewChange("install", "install a snap")
-	ts, err := snapstate.Install(s.state, "some-snap", "some-channel", 0)
+	ts, err := snapstate.Install(s.state, "some-snap", "some-channel", s.user.ID, 0)
```

Comment:
Let's have a auth field in fakeOp, and in the Download method of the fake backend we can then take the Macaroon out of the auther (`auther.(*store.Authenticator).Macaroon`), and insert on fakeOp. This way, we can check below that it is indeed set to "macaroon".

Without that, we're only testing that the SnapSetup is in place, I suspect, but not the logic that grabs the auther.